components/external_storage/examples/scli.rs

```diff
@@ -0,0 +1,161 @@
+use std::fs::File;
+use std::io::{Error, ErrorKind, Result};
+use std::path::Path;
+use std::sync::Arc;
+
+use clap::{App, ArgMatches};
+use external_storage::{
+    create_storage, make_local_backend, make_noop_backend, make_s3_backend, ExternalStorage,
+};
+use futures::executor::block_on;
+use futures_util::io::{copy, AllowStdIo};
+use ini::ini::Ini;
+use kvproto::backup::S3;
+
+static CMD: &str = r#"
```

Comment:
(I would have used [`structopt`](https://docs.rs/structopt). But this is not a big issue anyway.)